compliance-api/src/compliance_api/schemas/inspection_requirement.py

```diff
@@ -311,8 +311,9 @@ def nullify_nested(
         self, data, **kwargs
     ):  # pylint: disable=no-self-use, unused-argument
         """Make nested objects null if the referenced ID is null."""
-        data["req_type"] = {
-            "id": data.get("req_type").name,
-            "name": data.get("req_type").value,
-        }
+        if data.get("req_type") is not None:
```

Comment:
req_type should not be null. Is this condition to handle the existing data?

Reply:
<img width="1219" alt="Screenshot 2025-02-04 at 9 05 34 PM" src="https://github.com/user-attachments/assets/6b81ede3-427b-4546-baf6-5eda4f87dd97" />